docs/migration.md

```diff
@@ -18,17 +18,17 @@ Upgrading from v4.x to v5.0.0
 ### `qunit` and `@ember/test-helpers` dependencies
 
 Older versions of `ember-qunit` directly depended on `qunit` and
-`@ember/test-helpers`, in v5 this relationship was changed and now
-`ember-qunit` has `qunit` and `@ember/test-helpers` as peer dependencies.
+`@ember/test-helpers`. In v5, this relationship was changed and now
+`ember-qunit` has `qunit` and `@ember/test-helpers` (v2) as peer dependencies.
 
-In order to accomodate this change in your application you can run:
+In order to accomodate this change, in your application, you can run:
 
 ```sh
 # npm users
-npm install --save-dev qunit "@ember/test-helpers@^2.0.0-beta.4"
+npm install --save-dev qunit "@ember/test-helpers"
```

Comment:
The changes in this section are kinda outside of scope of what the pull request is about (documenting what to do for QUnit DOM).

I didn't think it'd be good to show installing the beta version when stable is now available. I can revert this change, or address it differently if needed.